Courses/DataScience/adoc/02_ExploratoryDataAnalysis.adoc

```diff
@@ -111,8 +107,8 @@ Which node label is the most popular one in this dataset?
 
 Select the correct answers.
 [%interactive]
-- [ ] [.false-answer]#Author#
-- [ ] [.required-answer]#Article#
+- [ ] [.false-answer]#Article#
+- [ ] [.required-answer]#Author#
```

Comment:
@ryguyrg seems like I had these the wrong way around. Not sure if we need to update the database to handle the new correct answer?